apps/omg_watcher_info/lib/omg_watcher_info/db/eth_event.ex

```diff
@@ -234,6 +237,69 @@ defmodule OMG.WatcherInfo.DB.EthEvent do
     {:ok, get(ethevent.root_chain_txhash_event)}
   end
 
+  # preload txoutputs in a single query as there will not be a large number of them
+  def get(root_chain_txhash_event) do
```

Comment:
you're mixing public and private functions. public on top, private on the bottom.

Reply:
Noted for future reference. Done e5513b8.